HW03POOL/POOL/Pool.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Threading;
+
+namespace MyPool;
+public class MyTask<TResult> : IMyTask<TResult>, ICalculatable   
+{
+    private Func<TResult> Calculation;
+    private bool isCompleted = false;
+	private TResult? result;
```

Comment:
```suggestion
private TResult? result;
```
форматирование поехало